addons/binding/org.openhab.binding.tankerkoenig/src/main/java/org/openhab/binding/tankerkoenig/internal/config/Station.java

```diff
@@ -30,7 +30,7 @@
     private String street;
     private String place;
     private String postCode;
-    private boolean isOpen;
+    private Boolean open;
```

Comment:
Did you check whether this break anything at runtime? I'm not completely familiar with the corner cases of GSON, I'm more the https://github.com/FasterXML/jackson-databind kind of guy 😄